extensions/amp-ima/0.1/validator-amp-ima.protoascii

```diff
@@ -0,0 +1,69 @@
+#
+# Copyright 2016 The AMP HTML Authors. All Rights Reserved.
```

Comment:
/cc @Gregable @powdercloud if they are ok with adding the validator changes here